### ISSUE TITLE
SecpR1 bit length 284?

### ISSUE THREAD

**Issue by tazle** [parallaxsecond/rust-psa-crypto#142]
In https://github.com/parallaxsecond/rust-psa-crypto/blob/7695a58a0a27fc6b8d2fd49a77f149fe987a3afb/psa-crypto/src/types/key.rs#L923C43-L923C46 the allowed SecpR1 bit lengths include 256, 284 and 521. Shouldn't the 284 be 384?

**Comment by hug-dev** [comment]
Yes you seem to be right this would be a typo! Unless it was a bug in previous version of PSA crypto/Mbed TLS!
